chain/client/src/stateless_validation/chunk_validator/mod.rs

```diff
@@ -640,6 +640,14 @@ impl Client {
         witness: ChunkStateWitness,
         processing_done_tracker: Option<ProcessingDoneTracker>,
     ) -> Result<(), Error> {
+        // Send the acknowledgement for the state witness back to the chunk producer.
+        // This is currently used for network roundtrip time measurement, so we do not need to
+        // wait for validation to finish.
+        if let Err(err) = self.send_state_witness_ack(&witness) {
+            tracing::warn!(target: "stateless_validation", ?err,
```

Comment:
You shouldn't need to use `Debug` (`?err`) to specify errors as a field, given that `impl tracing::Value for dyn std::error::Error` exists. Unfortunately I can't come up with a terser way to do it than this: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=da92d141e2430566ef42c7ba3f3967a5

One major benefit is that the tracing subscribers then get the entire error with all the causes, rather than just the top-level message which they can format in its entirety.

We should probably add some helpers to near-o11y at some point…